### DISCUSSION TITLE
Whether Electerm support this feature: when I type in a keyword, it automatically pops up a suggestion for I to choose from

### DISCUSSION THREAD

**Discussion by wilikeit** [electerm/electerm#3693] [Q&A]
Automatic prompt path

![image](https://github.com/user-attachments/assets/07871430-d0f6-4b89-9c67-6f74b57a9a27)

Automatic prompt command
![image](https://github.com/user-attachments/assets/74e14dd0-7cbc-497b-9ab9-2c5d13170276)

I can't find the relevant Settings in electerm.

**Comment by zxdong262** [comment]
https://github.com/electerm/electerm/issues/3694 Created a issue to track this
